src/provideCore.ts

```diff
@@ -24,10 +24,8 @@ export function provideCore(config: SearchConfig): SearchCore {
 
   const defaultedConfig: SearchConfigWithDefaulting = {
     ...config,
-    endpoints: {
-      ...defaultEndpoints,
-      ...config.endpoints
-    }
+    endpoints:
+      new EndpointsProvider(Environment.PROD, CloudRegion.US, config.endpoints).getEndpoints()
```

Comment:
taking the config object would simplify this

Reply:
No longer relevant. Ack